src/main/kotlin/com/github/avrokotlin/avro4k/schema/SchemaFor.kt

```diff
@@ -39,7 +39,7 @@ class EnumSchemaFor(
    private val descriptor: SerialDescriptor
 ) : SchemaFor {
    override fun schema(): Schema {
-      val naming = RecordNaming(descriptor)
+      val naming = RecordNaming(descriptor, DefaultNamingStrategy)
```

Comment:
Why the DefaultNamingStrategy?

Reply:
I guess this actually should never call the naming strategy or this is the enum name. And probably you don't want to touch it? Or if change this  to use given strategy then example snake_casing will generate enum looking like this: 

``` 
{
  "name" : "v",
  "type" : {
    "type" : "enum",
    "name" : "_internet_protocol",
    "namespace" : "com.github.avrokotlin.avro4k.schema",
    "symbols" : [ "IPv4", "IPv6" ]
  }
}
```

Field level name v is okay but that name object probably should not be touched? Well I don't know how exactly python / etc components actually use those schemas to generates types.. Or does it use some kind of dict structure for avro objects dunno